core/src/avm1/globals/system.rs

```diff
@@ -59,9 +62,26 @@ impl fmt::Display for SandboxType {
     }
 }
 
+/// Type of the Ruffle player
+pub enum RuffleType {
+    DesktopPlayer,
+    WebPlayer,
```

Comment:
Naming: I might prefer simply `Desktop` and `Web`.

Reply:
While I prefer it this way, I don’t have a strong opinion about it, so I can change it if you want.

Reply:
I think I prefer the shorter versions for brevity (would there ever be a `RuffleType` that isn't a `Player`?)

Reply:
Okay, I'll change it :)